react-effects/src/index.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom/client';
+import readItems from './read.js';
+
+function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    readItems()
+      .then(value => {
+        setIsLoading(!isLoading);
+        setItems(value);
+      })
+      .catch(err => {
+        console.error(err.message);
+      });
+  }, [isLoading]);
```

Comment:
Because of this dependency, this effect will run again once `isLoading` changes to false on line 13. Once you remove the dependency on line 13, you will be able to pass `[]` so that the effect will run only once, when the component is mounted.